optuna/pruners/hyperband.py

```diff
@@ -24,6 +24,21 @@ class HyperbandPruner(BasePruner):
     Note that this implementation does not take as inputs the maximum amount of resource to
     a single SHA noted as :math:`R` in the paper.
 
+    .. note::
+        If you use ``HyperbandPruner`` with :class:`~optuna.samplers.TPESampler`,
+        it's recommended to consider to set larger ``n_trials`` or ``timeout`` to make full use of
+        the characteristics of :class:`~optuna.samplers.TPESampler`
+        because :class:`~optuna.samplers.TPESampler` uses some (by default, :math:`10`)
+        :class:`~optuna.trial.Trial`\s for its startup.
```

Comment:
```suggestion
        :class:`~optuna.trial.Trial` for its startup.
```

flake8 failed here.
```
#!/bin/bash -eo pipefail
. venv/bin/activate
flake8 .
./optuna/pruners/hyperband.py:32:37: W605 invalid escape sequence '\s'
1     W605 invalid escape sequence '\s'
```
https://circleci.com/gh/optuna/optuna/25986

Reply:
Sorry my previous suggest was violating flake8. You can also write as follows to keep the same result. 
```
:class:`~optuna.trial.Trial`\\ s for its startup.
```